Src/BootCamp.Chapter/Program.cs

```diff
@@ -2,11 +2,95 @@
 
 namespace BootCamp.Chapter
 {
-    class Program
+    public class Program
     {
-        static void Main(string[] args)
+        private static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            ConvertFromDecimalToBinary();
+            ConvertFromBinaryToDecimal();
+            PrintEmicon();
+        }
+
+        private static void PrintEmicon()
+        {
+            Console.OutputEncoding = System.Text.Encoding.Unicode;
+            Console.WriteLine("Give a a W, A, S or D");
+            var input = Console.ReadLine().ToUpper();
+            var output = ""; 
+            switch (input)
+            {
+                case "W":
+                    output = "↥";
+                    break;
+
+                case "A":
+                    output = "↤";
+                    break;
+
+                case "S":
+                    output = "↧";
+                    break;
+
+                case "D":
+                    output = "↦";
+                    break;
+
+                default:
+                    output = "This is not a valid character. YOu must choose between w,a,s or d";
+                    break; 
+            }
+
+            Console.WriteLine(output);
+        }
+
+        private static void ConvertFromBinaryToDecimal()
+        {
+            Console.Write("Which  do you want to convert to binary: ");
+            var input = Console.ReadLine();
+            try
+            {
+                var binaryNumber = int.Parse(input);
+                var realNumber = binaryNumber;
+                int decimalValue = 0;
```

Comment:
This is not a decimalValue

Reply:
??  in my mind its the outcome of the converting a binary number to a decimal number. 
So what is it then ?

Reply:
Oke, I came up with a new name. I hope it's good now